CONTRIBUTING.md

```diff
@@ -0,0 +1,130 @@
+# Contributing to the Cordova Showcase Application Template
+
+The Cordova Showcase Application Template is part of the [AeroGear project](https://aerogear.org/), see the [Community Page](https://aerogear.org/community) for general guidelines for contributing to the project.
+
+This document details specifics for contributions to the Cordova Showcase Application Template.
+
+## Issue tracker
+
+The tracking of issues for the AeroGear Cordova Showcase Application Template is done in the [AeroGear Android Project](https://issues.jboss.org/projects/AEROGEAR/issues) in the [JBoss Developer JIRA](https://issues.jboss.org).
+
+See the [AeroGear JIRA Usage and Guidelines Guide](https://aerogear.org/docs/guides/JIRAUsage/) for information on how the issue tracker relates to contributions to this project.
+
+## Asking for help
+
+Whether you're contributing a new feature or bug fix, or simply submitting a
+ticket, the Aerogear team is available for technical advice or feedback. 
+You can reach us at [#aerogear](ircs://chat.freenode.net:6697/aerogear) on [Freenode IRC](https://freenode.net/) or the 
+[aerogear google group](https://groups.google.com/forum/#!forum/aerogear)
+-- both are actively monitored.
+
+## Prerequisites
+
+Ensure you have the following installed in your machine:
+
+- [Node](https://nodejs.org/en/) - version 8.9.4
+- [Ionic](https://ionicframework.com/) - version 3.20.0
+- [Cordova](https://cordova.apache.org/) - version 8.0.0
+
+
+# Developing the Cordova Showcase Application Template for testing new SDK features
+This application is intended to showcase the latest release of [Aerogear Js SDK](https://github.com/aerogear/aerogear-js-sdk), so pre-release work should never be merged to the `master` branch, any changes made pre-release can be merged to the `development` branch only.
```

Comment:
I think after discussion with @wtrocki on the mailing list we can remove this section now. Instead we can add it to the showcasing section of the readme to just checkout the desired git tag for whatever sdk version and run that instead.

Reply:
I will look for more comments on this email thread to confirm that everyone is happy with that aproach, but yes let's not include this. Using standard open source flow do not require any documentation. Things are just merged to default branch (usually master)